proto/desmos/posts/v3/genesis.proto

```diff
@@ -14,14 +15,28 @@ message GenesisState {
   option (gogoproto.goproto_stringer) = true;
 
   repeated SubspaceDataEntry subspaces_data = 1
-      [ (gogoproto.nullable) = false ];
-  repeated PostDataEntry posts_data = 2 [ (gogoproto.nullable) = false ];
-  repeated Post posts = 3 [ (gogoproto.nullable) = false ];
-  repeated Attachment attachments = 4 [ (gogoproto.nullable) = false ];
-  repeated ActivePollData active_polls = 5 [ (gogoproto.nullable) = false ];
-  repeated UserAnswer user_answers = 6 [ (gogoproto.nullable) = false ];
-  Params params = 7 [ (gogoproto.nullable) = false ];
-  repeated PostOwnerTransferRequest post_owner_transfer_requests = 8 [ (gogoproto.nullable) = false ];
+      [ (gogoproto.nullable) = false, (amino.dont_omitempty) = true ];
```

Comment:
Cosmos-SDK adds amino annotation inside genesis files as well.
https://github.com/cosmos/cosmos-sdk/blob/ab77fe20d3c00ef4cb73dfd0c803c4593a3c8233/proto/cosmos/bank/v1beta1/genesis.proto#L15

But, I think genesis is not needed to support Amino. What do you think? @RiccardoM

Reply:
I agree it should not be added to the genesis file. However, we can keep it for now as it's only metadata that does not truly affect anything to be honest.